lib/fake_data_loader.rb

```diff
@@ -62,8 +62,20 @@ def self.load_sessions
           :feedback_session,
           provider: user,
           receiver: User.where.not(id: user.id).sample
-        )
+        ).tap do |session|
+          create(
+            :comment,
+            feedback_session: session,
+            body: Faker::Lorem.paragraph(sentence_count: 2, supplemental: true, random_sentences_to_add: 4)
+          )
+          create(
+            :comment,
+            feedback_session: session,
+            body: Faker::Lorem.paragraph(sentence_count: 2, supplemental: true, random_sentences_to_add: 4)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/LineLength: Line is too long. [107/100]